tests/cli/test_datasets.py

```diff
@@ -1400,12 +1400,54 @@ def test_add_same_filename_multiple(runner, client, directory_tree):
     result = runner.invoke(
         cli, [
             'dataset', 'add', '--force', 'my-dataset1', directory_tree.strpath,
-            'Dockerfile'
+            'README.md'
```

Comment:
Will github/gitlab show readme.md in case renku moves it to LFS?

Reply:
In case it breaks it, maybe we should add this to the blacklist as well.

Reply:
Probably it will just show a `The source could not be displayed because it is stored in LFS. You can download it instead. ` style message.

But this is not a file that's needed for any process/workflow/ci/what have you, so it doesn't affect renku. So if users want to put it on LFS, I don't see a reason to stop them.

Reply:
Sure my point is more aiming at protecting users from themselves and crazy `accidents` people execute on the shell. I agree that there is no need to hardcode blacklisting of such files, maybe we should still think about putting such files to (future) `.renkuignore` which could be related to template important files. One of such files there maybe it could be `README.md`